test/spec.dump

```diff
@@ -1,3 +1,897 @@
 {
-"annotations": {"io.container.manager": "libpod"}
+  "ociVersion": "1.0.2-dev",
```

Comment:
I would prefer if you only add one or two mounts. Just the minimum we need for tests. Nothing else.